lib/testscript_engine/testscript_runnable.rb

```diff
@@ -176,14 +177,17 @@ def load_fixtures
   end
 
   def load_profiles
+    puts ("      Loading profiles...")
     script.profile.each do |profile|
       next warning(:no_static_profile_id) unless profile.id
       next warning(:no_static_profile_reference) unless profile.reference
 
       profile_server = FHIR::Client.new("")
       response = profile_server.send(:get, profile.reference, { 'Content-Type' => 'json' })
```

Comment:
would be nice to support a pointer to a file rather than a URL to use to load the structure definition, that way you could validate against profiles under development

Reply:
Done : https://github.com/fhir-crucible/testscript-engine/pull/70/commits/0d028a191c018d387b21e8301c957af7ba5ef3db